Makefile

```diff
@@ -0,0 +1,39 @@
+PACKAGE_NAME=osbuild
+VERSION=1
```

Comment:
We probably want to use `git describe` here, but we can do that in a follow-up.

Reply:
ok